src/js/modifiers.js

```diff
@@ -275,23 +275,24 @@ function modify_rotate() {
   console.log(transposer)
   console.log(equave)
 
-  // transpose each line, mod equave
-  for (let i = 0; i < lines.length; i++) {
-    if (i !== degree) {
-      lines[i] = moduloLine(transposeLine(lines[i], transposer), equave)
-    } else {
-      lines[i] = equave
-    }
-    console.log(i)
+  let rotatedLines = []
+
+  // transpose lines, mod equave
+  // start on degree after transposer/new root, cycle around and stop before transposer
+  for (let i = 1; i < lines.length; i++) {
+    let deg = (i + degree) % lines.length
+    let index = i - 1
+    rotatedLines[index] = moduloLine(transposeLine(lines[deg], transposer), equave)
+    console.log(index)
     console.log(degree)
```

Comment:
I would recommend cleaning up `console.log` calls once the development of a feature have been finished.